scripts/get_git

```diff
@@ -45,8 +45,10 @@ build_msg "CLR_GET" "GIT FETCH" "${1}"
 git -C "${PACKAGE}" fetch --depth 1 origin ${PKG_VERSION}
 git -C "${PACKAGE}" checkout FETCH_HEAD
 
-build_msg "CLR_GET" "GIT SUBMODULE" "${1}"
-git -C "${PACKAGE}" submodule update --init --recursive --depth 1
+if [ "${GET_SKIP_SUBMODULE}" != "yes" ]; then
+  build_msg "CLR_GET" "GIT SUBMODULE" "${1}"
+  git -C "${PACKAGE}" submodule update --init --recursive --depth 1
+fi
```

Comment:
OK, so here is the addtion for GET_SKIP_SUBMODULE... why?

Reply:
Because setting depth on that package breaks building, So I added an option, so I can pull submodules in package, and fix the issue. It is a hack workaround for coreboot sources, where I get cbfstool from.